### PR TITLE
Add golang symbols detection in aaa Fix #10959 ##anal

### LINKED ISSUE #10959: Integrate Go-Helpers natively
Integrate Go-Helpers from zlowram natively, https://github.com/zlowram/radare2-scripts/tree/master/go_helpers.

According to telegram user, the script need to be updated by changing the size calculation

### PR DESCRIPTION
**Pull request by Maijin**

### CODE REVIEW

**Comment by Maijin** [comment]
The binary has only 2100 symbols detected.

I don't understand how comes `aa` is faster than `af @@ sym.*` but is supposed to do the same according to the help, this can also be observed with `aa*` vs `aa` time.

**Comment by radare** [comment]
please rebase

File: libr/core/cmd_anal.c [hunk 1]
```diff
@@ -6826,7 +6826,7 @@ static void rowlog(RCore *core, const char *str) {
 	}
 }
 
-static void rowlog_done(RCore *core) {
+R_API void rowlog_done(RCore *core) {
```
Comment (radare) [inline libr/core/cmd_anal.c]: dont make non r_* apis public with R_API, this should be static, or refactored into r_print

**Comment by radare** [comment]
can you share the binary?

**Comment by Maijin** [comment]
@radare - Sure it's https://github.com/radare/radare2-regressions/blob/master/bins/elf/analysis/go_stripped2

**Comment by Maijin** [comment]
But the aa*, aa analysis difference can be seen using `/bin/ls` or any binary

**Comment by Maijin** [comment]
For reference, I'm now using aF instead of af which does the trick for this case

File: libr/util/print.c [hunk 2]
```diff
@@ -1968,3 +1968,30 @@ R_API void r_print_hex_from_bin (RPrint *p, char *bin_str) {
...
+
+R_API char* r_print_rowlog(RPrint *print, char *str) {
+	int use_color = print->flags & R_PRINT_FLAGS_COLOR;
+	bool verbose =  print->scr_prompt;
+	char* oldstr = str;
+	if (!verbose) {
+		return oldstr;
+	}
+	if (use_color) {
+		eprintf ("[ ] "Color_YELLOW"%s\r["Color_RESET, str);
```
Comment (xarkes) [inline libr/util/print.c]: Why `eprintf` here?
Isn't eprintf only used to print debug info on `stderr`?

Comment (Maijin) [reply libr/util/print.c]: This was like that before probably for a reason

**Comment by Maijin** [comment]
I'm rebasing because tests got executed after merge of tests

**Comment by radare** [comment]
the pr lgtm, the time diff between aa and aa* should be tracked in a separate issue

File: libr/util/print.c [hunk 3]
```diff
@@ -1968,3 +1968,30 @@ R_API void r_print_hex_from_bin (RPrint *p, char *bin_str) {
 	p->cb_printf ("\n");
 	free (buf);
 }
+
+R_API char* r_print_rowlog(RPrint *print, char *str) {
```
Comment (radare) [inline libr/util/print.c]: use const char *

File: libr/util/print.c [hunk 4]
```diff
@@ -1968,3 +1968,30 @@ R_API void r_print_hex_from_bin (RPrint *p, char *bin_str) {
...
+	int use_color = print->flags & R_PRINT_FLAGS_COLOR;
+	bool verbose =  print->scr_prompt;
+	char* oldstr = str;
```
Comment (radare) [inline libr/util/print.c]: use const char *

File: libr/util/print.c [hunk 5]
```diff
@@ -1968,3 +1968,30 @@ R_API void r_print_hex_from_bin (RPrint *p, char *bin_str) {
...
+	}
+	if (use_color) {
+		eprintf ("[ ] "Color_YELLOW"%s\r["Color_RESET, str);
+	} else {
+		eprintf ("[ ] %s\r[", str);
+	}
+	return oldstr;
+}
+
+R_API void r_print_rowlog_done(RPrint *print, char *oldstr) {
```
Comment (radare) [inline libr/util/print.c]: use const char *

**Comment by radare** [comment]
please fix the comments

**Comment by Maijin** [comment]
Fixed!

**Pull request merged**